src/basho_bench_config.erl

```diff
@@ -51,32 +68,69 @@
 ensure_started() -> 
     start_link().
 
+
 start_link() ->
     gen_server:start_link({global, ?MODULE}, ?MODULE, [], []).
 
 
 load(Files) ->
     ensure_started(),
     gen_server:call({global, ?MODULE}, {load_files, Files}). 
-    
+
+
```

Comment:
In that case it IS following the whitespace style of the neighboring functions, was just inconsistent